### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by TheBigLee**
Signed-off-by: Nicolas Bigler <nicolas.bigler@vshn.ch>

## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

File: postprocess/update_images.jsonnet [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  'create-certificate-job',
+  'set-crd-version-job',
+  'wait-job',
+  'wait-webhooks-job',
+];
+
+local image = '%s/%s:%s' % [ params.images.kubectl.repository, params.images.kubectl.image, params.images.kubectl.tag ];
+
+{
+  [file]: com.yaml_load(std.extVar('output_path') + '/' + file + file_extension) {
```
Comment (zugao) [inline postprocess/update_images.jsonnet:19]: what is the reason we are changing the image version from the default version of the chart?

**Review by zugao — APPROVED** [review]
Looks good so far, I haven't tested though. I have just one question regarding image version in some resources.

File: postprocess/update_images.jsonnet (hunk 1 above)
Comment (susana-garcia) [reply postprocess/update_images.jsonnet:19]: so, we had an issue downloading the image (pull rate limit) and decided to make a post processing to point it to our cache.

File: class/defaults.yml [hunk 2]
```diff
@@ -2,3 +2,12 @@ parameters:
   stackgres_operator:
     =_metadata: {}
     namespace: syn-stackgres-operator
+    charts:
+      stackgres-operator: 1.4.0
```
Comment (glrf) [inline class/defaults.yml]: Please use the following structure for chart dependencies: https://syn.tools/syn/explanations/commodore-components/helm-charts.html

That way our renovate will be able to update it

File: class/defaults.yml [hunk 3]
```diff
@@ -2,3 +2,12 @@ parameters:
...
+    helmValues:
+      deploy:
+        restapi: false
+    images:
+      kubectl:
+        image: ongres/kubectl
+        tag: v1.24.3-build-6.16
```
Comment (glrf) [inline class/defaults.yml]: You don't set a `repository` in the defaults. That means if we deploy this component without  setting a repository you'll patch the image to `/ongres/kubectl:v1.24.3-build-6.16` which is AFAIK not valid.  Either set a default repository as `docker.io` or handle the missing repository in the post processing.

**Review by glrf — CHANGES_REQUESTED** [review]
LGTM overall, but there are two issues that still need to be fixed

**Review by glrf — APPROVED** [review]
LGTM now

**Pull request merged**